### DISCUSSION TITLE
Same CFG, Same Seed Lower Quality Than AUTOMATIC1111

### DISCUSSION THREAD

**Discussion by ertugrul-dmr** [vladmandic/sdnext#469] [General]
Is there setting for getting deterministic and similar results AUTOMATIC1111. Or this case is specific to me? I am using same cfg and models with both repos and getting faster but much worse results...

**Comment by vladmandic** [comment]
Are you sure it's same CFG?
First check Cross-optimization and Cuda settings.
Second, generate image wiyh A1111 and import it here to read it's params instead of trying to set each manually.

**Reply by ertugrul-dmr** [reply]
Alright, got 99.9 similar results after disabling cross-optimization, hires fix got slower than automatic (like %20 in a single example). But anyways thank you! Will check cuda settings too...
